tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,14 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    # raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
Nice! One thing to consider, this assertion expects items in that specific order -- if we wanted to make our test more flexible, we can opt for something like: 
```
assert item_a in tai.inventory and item_b in tai.inventory and item_f in tai.inventory
assert item_d in jesse.inventory and item_e in jesse.inventory and item_c in jesse.inventory
```